CONTRIBUTING.md

```diff
@@ -0,0 +1,61 @@
+# Contributing to theupdateframework.io
+We're excited that you're interested in contributing to the project! 🎉 Your contributions are highly valued and welcome. 🎉
+
+## Issues & Pull Requests
+
+### Creating an Issue
+
+Before **creating** an Issue i.e for `bug`/`features`/`improvements` please follow these steps:
+
+
+1.Check Existing Issues: Search to see if the issue already exists.
+
+2.Provide Context: If it doesn't exist, create a new issue with as much detail as possible. Make sure to select the correct issue type (e.g. bug,documentation,feature).
+
+3.Express Interest: If you want to work on the issue after it's been reviewed, mention this in your issue description.
+
+### Working on an Issue
+
+Before starting work on an existing issue, please follow these steps:
+
+1. **Request Assignment**: Comment on the issue asking for it to be assigned to you.
+2. **Prepare for Assignment**:
+    - Confirm that you have read the `CONTRIBUTING.md`.
+    - Ensure you have a functional development environment (you can build and run the project).
+    - Describe your intended approach to solving the issue.
+    - Include these points in one or more comments.
+3. **Start Work**: Begin working on the issue only after it has been assigned to you.
+4. **Avoid Conflicts**: Only start working on the issue and open a Pull Request after it has been assigned to you. This prevents confusion, duplicate work, and potential conflicts among contributors.
+5. **Reference the Issue**: In your Pull Request, reference the issue (e.g., `This PR fixes #1234`). This ensures the issue is automatically closed when your Pull Request is merged.
+
+
+> **Notes:**
+>
+> - **Check the Assignees Box**: Before requesting to be assigned to an issue, check the `Assignees` box at the top of the page to see if it has already been assigned to someone else. If it has a current assignee but appears to be inactive, politely ask them if they are still working on it or if you might collaborate with them.
+> - **Requesting Assignment**: Only request to be assigned to an issue if you know how to work on it.
+> - **Seek Clarity**: If an issue is unclear, ask questions to get more information before requesting to be assigned. Avoid asking vague questions like "what do I do next?" or "how do I fix this?"
+> - **Collaboration**: An issue can be assigned to multiple people if all parties agree to collaborate on it. The pull request can contain commits from different collaborators.
+> - **Inactivity**: Any issue with no activity for 2 weeks will be unassigned and re-assigned to someone else.
+
```

Comment:
IMO we should add `DCO` section clearly in `CONTRIBUTING.md`. In every CNCF project it is included see [here ](https://theupdateframework.readthedocs.io/en/latest/CONTRIBUTING.html#dco)in `python-tuf`, just for [reference](https://github.com/cncf/foundation/blob/main/dco-guidelines.md#dco-guidelines-v10). WYT? @Ayush9026 
Any suggestions @lukpueh @joshuagl

Reply:
That's a great point. Including a DCO section would align with CNCF project standards and provide clarity for contributors. I’ll add this section to the CONTRIBUTING.md once @lukpueh and @joshuagl sir have reviewed it.

Thank you for the suggestion! 🙏